src/squlearn/kernel/matrix/projected_quantum_kernel.py

```diff
@@ -465,7 +466,8 @@ def get_params(self, deep: bool = True) -> dict:
         Return:
             Dictionary with hyper-parameters and values.
         """
-        params = dict(measurement=self._measurement_input)
+        params = super().get_params(deep=False)
+        # params = dict(measurement=self._measurement_input)
```

Comment:
again unnecessary comment
```suggestion
        params = super().get_params(deep=False)
```